src/rgw/librgw.cc

```diff
@@ -489,6 +491,8 @@ namespace rgw {
     rgw_tools_init(g_ceph_context);
 
     rgw_init_resolver();
+    rgw::curl::setup_curl(boost::none);
+    rgw_http_client_init(g_ceph_context);
```

Comment:
this must be dropped, as this was mimic introduced, waiting for jenkins to confirm

Reply:
does not pass "make check" - @theanalyst Is that the confirmation you were looking for?